e2e/scenario_test/settings/not_set/correction_range/settings.no_correction_range.spec.js

```diff
@@ -1,22 +1,16 @@
-const { setup, pump, cgm, match, settings } = require('../../src/index');
+const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
 
 describe('Pump Settings', () => {
     beforeAll(async () => {
         await setup.LaunchLoop();
         await setup.LoadScenarios(device.id);
         await cgm.Add();
         await pump.Add();
-        await setup.LoadScenario('flat_cgm');
-    });
-    afterAll(async () => {
-        await cgm.Remove();
-        await pump.Remove();
     });
     describe('Closed loop is not allowed', () => {
         describe('When correction range is not set', () => {
-            afterAll(async () => {
-                await cgm.RemoveData();
-                await pump.RemoveData();
+            beforeAll(async () => {
+                await setup.LoadScenario('flat_cgm');
```

Comment:
[?] I'm confused why there are two `setup.LoadScenario` calls, one here and one in `beforeAll`.